packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -98,19 +98,21 @@ class GlowingOverscrollIndicator extends StatefulWidget {
   _GlowingOverscrollIndicatorState createState() => new _GlowingOverscrollIndicatorState();
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    description.add('$axisDirection');
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(new EnumProperty<AxisDirection>('axisDirection', axisDirection));
+    String showMessage;
     if (showLeading && showTrailing) {
-      description.add('show: both sides');
+      showMessage = 'both sides';
     } else if (showLeading) {
-      description.add('show: leading side only');
+      showMessage = 'leading side only';
     } else if (showTrailing) {
-      description.add('show: trailing side only');
+      showMessage = 'trailing side only';
     } else {
-      description.add('show: neither side (!)');
+      showMessage = 'neither side (!)';
     }
-    description.add('$color');
+    description.add(new StringProperty('show', showMessage));
```

Comment:
Is the desired display the property like
`show: both sides` as matches the previous debugging output
 or was that an error and the message like
`show both sides` more consistent with other diagnostic output?
If `show both sides` is preferred then `Diagnostics.message` is the right choice. If `show: both sides` is desired object display in IntelliJ will look most consistent if you use a diagnostic property any time you want to write `name: value`
because in IntelliJ the name, value pairs won't be displayed with `:` as the separator. For example, property names will definitely be displayed in a different color than values and a separator of `=` might be used for consistency with other IntelliJ debugger views. Eg, https://d3nmt5vlzunoa1.cloudfront.net/idea/files/2015/02/decompiler_and_debugger.png

Note that if the property view is desired, I'd argue the most technically correct but somewhat overkill thing to do would be to write:
description.new DiagnosticProperty<OverflowSides>('show', new OverflowSides(showLeading: showLeading, showTrailing: showTrailing))
where OverflowSides has a toString that implements the showMessages described above.
As a half was measure to that option I changed the name showMessage to showDescription to make it clearer it is a description of the property value for 'show'.